### PR TITLE
 First Page of 1907 March 18

### PR DESCRIPTION
**Pull request by FlameWo**

### CODE REVIEW

File: 1907-03-18.xml [hunk 1]
```diff
@@ -0,0 +1,1583 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/dig-eg-gaz/resources/master/out/egSchema.rnc" type="application/relax-ng-compact-syntax"?>
+<?xml-stylesheet type="text/xsl" href="dig-eg-gaz.xsl"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor></editor>
```
Comment (whanley) [inline 1907-03-18.xml]: add your name as editor

File: 1907-03-18.xml [hunk 2]
```diff
@@ -0,0 +1,1583 @@
...
+                                <cell>Leaves Alexandria</cell>
+                                <cell><date when="1907-12-08">December 8 &amp; 22</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1907-12-11">Decmber 11 &amp; 25</date> a.m.</cell>
+                                <cell>arr. Genoa</cell>
+                                <cell><date when="1907-12-12">Decmber 12 &amp; 26</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1908-01-12">January 12,19 &amp; 26</date> ,,</cell>
```
Comment (whanley) [inline 1907-03-18.xml]: this and other dates in this ad should read 1908

Comment (whanley) [reply 1907-03-18.xml]: I mean 1907

File: 1907-03-18.xml [hunk 3]
```diff
@@ -0,0 +1,1583 @@
...
+                            </row>
+                            <row>
+                                <cell>Le Jeudi</cell>
+                                <cell>7 et Vendredi 22</cell>
+                                <cell>à 5 h. p.m.</cell>
+                                <cell>pour Port-Saïd.</cell>
+                            </row>
+                        </table>
+                        <p><measure type="indexNo">31-12-906</measure></p>
+                    </div> <!--Could not be located (Advertisement)-->
```
Comment (whanley) [inline 1907-03-18.xml]: did you submit an issue for this missing ad?

**Comment by FlameWo** [comment]
I am certain I did place my name in the second one and I followed the
dates that were on the paper, I could send you my image for the page.

File: 1907-03-18.xml [hunk 4]
```diff
@@ -0,0 +1,1584 @@
...
+                                <cell><date when="1907-01-12">January 12,19 &amp; 26</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1907-01-15">January 15,22 &amp; 29</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell>...........</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1907-02-02">February 2,9,16 &amp; 23</date> ,,</cell>
+
```
Comment (whanley) [inline 1907-03-18.xml]: when = "1907-04-17"

File: 1907-03-18.xml [hunk 5]
```diff
@@ -0,0 +1,1584 @@
...
+                        <p>For details and illustrated programmes apply to HEAD OFFICE Sharia Boulac, 
+                           Continental Hotel Buildings. CAIRO</p>
+                        <p><title>Hamburg-Amerika Linie.</title></p>
+                        <p>Accelerated Service by:<lb/> S.S. Oceana of 8,000 Tons, 350 berths including 90
+                            single cabins, greatest comfort, best cuisine.</p>
+                        <p>FARES: to Naples from £10, to Genoa from £13.</p>
+                        <table>
+                            <row>
+                                <cell>Leaves Alexandria</cell>
+                                <cell><date when="1907-12-08">December 8 &amp; 22</date> p.m.</cell>
```
Comment (whanley) [inline 1907-03-18.xml]: These three December dates are all 1906, not 1907.

File: 1907-03-18.xml [hunk 6]
```diff
@@ -0,0 +1,1584 @@
...
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1907-03-18"><name>Monday</name>, March 18, 1907</date>
+                    <extent><measure unit="pages" quantity="8"></measure> pages</extent>
```
Comment (whanley) [inline 1907-03-18.xml]: add an 8 before pages

File: 1907-03-18.xml [hunk 7]
```diff
@@ -0,0 +1,1584 @@
...
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    
+                    <head type="sub">No. <measure quantity="7718">7,718</measure>] ALEXANDRIA, MONDAY, MARCH 18, 1907. [ EIGHT PAGES P.T. 1.]
+                    </head>
+                </div>
+                <div type="section" scope="advertisement"> 
+                    <cols n="3"/>
+                    <cb n="1"/>
+                    <div type="advert" xml:id="deg-ad-etc01">
```
Comment (whanley) [inline 1907-03-18.xml]: this ad should be in the masthead division (above)

File: 1907-03-18.xml [hunk 8]
```diff
@@ -0,0 +1,1584 @@
...
+                        <head>The Eastern Telegraph Company, Limited.</head>
+                        <p>This Company's system of submarine telegraph <lb/>cables is the most direct and
+                            quickest means of <lb/>communication from Egypt to Europe, North and <lb/>South America,
+                            East, South and West Africa, <lb/>India, Australia, New Zealand, China and Japan.</p>
+                        <p>To secure quick transmission, telegrams should <lb/>be marked <hi>Via Eastern.</hi></p>
+                        <p>For latest average time to London, see daily <lb/>bulletin in this paper.</p>
+                        <p>STATIONS IN EGYPT: Alexandria, Cairo, <lb/>Suez, Port-Tewfik, Port-Saïd, Suakin. Head
+                            <lb/>Office. London.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-sfp01">
```
Comment (whanley) [inline 1907-03-18.xml]: this ad should be in the masthead division (above)

File: 1907-03-18.xml [hunk 9]
```diff
@@ -0,0 +1,1584 @@
...
+                    <p>The issue of return tickets to and from Egypt has been discontinued.
+                        Passengers paying full fare one direction will, however be allowed sbatement
+                        of one third off fare back, if return voyage be made within four months of
+                        arrival or abatement of £ 0 o/o if return was made within six months of
+                        arrival.</p>
+                    <p>Special Rate during Summer season 15th May - 15 September.</p>
+                    <p>Agents. Cairo:—Thos. Cook &amp; Son. Alexandria : —R. J. Moss &amp; Co.—For
+                        all particulars apply to Wm. STAPLEDON &amp; Sons, PORT SAID and PORT-TEWFIK
+                        (Suez). 31-12-906</p>
+                    <div type="advert" xml:id="deg-ad-blm02"> <!--Am not sure of what the quantity or the amount for the weight would be.-->
```
Comment (whanley) [inline 1907-03-18.xml]: this ad should be preceded by a `</div>` tag, closing the previous ad.

File: 1907-03-18.xml [hunk 10]
```diff
@@ -0,0 +1,1584 @@
...
+                        <p>Throw Steel Nibs Away!</p>
+                        <p>They waste time, are costly to use, wear out, scratch, etc.</p>
+                        <p>Try a "Swan" Fountain Pen.</p>
+                        <p>Prices from 10/6</p>
+                        <p>Sold by all Stationers and Importers in Cairo and Alexandria.</p>
+                        <p>Catalogue free.</p>
+                        <p>Mable, Todd &amp; Bard, High Holborn, London.</p>
+                    </div>
+                </div>
+                <div type="advert" xml:id="deg-ad-orm01">
```
Comment (whanley) [inline 1907-03-18.xml]: this and all of the ads below should go in the `<div type="section">` above.

File: 1907-03-18.xml [hunk 11]
```diff
@@ -0,0 +1,2174 @@
...
+                                    <cell>Bombay &amp; Karachi</cell>
+                                    <cell><name>S.S. Trafford Hall</name></cell>
+                                    <cell><date when="1907-04-17">April 17</date></cell>
+                                </row>
+                            </table>
+                            <p>SALOON FARES:—Port Said to Malta £4.10.0. Marseilles. £10.0.0. London
+                                or Liverpool, £l4.l0.0. Colombo, Calcutta, Bombay or Karachi,
+                                £25.0.0. Special rates for steamers not carrying Doctor or
+                                Stewardess. For further particulars apply to</p>
+                            <p>CORY BROS. &amp; Co., Ltd., Agents for CITY Line, Port Sa
```
Comment (whanley) [inline 1907-03-18.xml]: 1907

File: 1907-03-18.xml [hunk 12]
```diff
@@ -0,0 +1,2174 @@
...
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head type="sub">No. <measure quantity="7718">7,718</measure>] ALEXANDRIA,
+                        MONDAY, MARCH 18, 1907. [ EIGHT PAGES P.T. 1.] </head>
+                    <div type="section" scope="advertisement">
```
Comment (whanley) [inline 1907-03-18.xml]: add a `</div>` to close the masthead, and put the first two ads below (which appear on either side of the title) into the masthead.

File: 1907-03-18.xml (hunk 11 above)
Comment (FlameWo) [reply 1907-03-18.xml]: ?

Comment (whanley) [reply 1907-03-18.xml]: Sorry, I meant change the date at line 1115, which reads `<cell><date when="1908-04-17">April 17</date> ,,</cell>`

**Comment by FlameWo** [comment]
Hi, I just received this from the company today. I had my Wednesday page
saved on the computer, though it seems that it will not be back in time; I
wanted to ask if I could send it to you after Friday. I planned on it
taking a week but it seems as though it will be longer for it to be
repaired, I sent it during March.
---------- Forwarded message ----------
From: <hp.customer.care@hp.com>
Date: Thu, May 3, 2018 at 4:36 PM
Subject: Your HP Product has been received and repair is in process
To: HANDYP12@gmail.com


[image: HP Logo] support.hp.com <http://www.support.hp.com>
------------------------------
Customer Product Received
Customer Service Order # BLPN0743-01
HANDY PIERRE
322 STADIUM DR
<https://maps.google.com/?q=322+STADIUM+DR+%0D%0A+TALLAHASSEE,++FL++32304&entry=gmail&source=g>
TALLAHASSEE, FL 32304
<https://maps.google.com/?q=322+STADIUM+DR+%0D%0A+TALLAHASSEE,++FL++32304&entry=gmail&source=g>




*Important: *** PLEASE DO NOT REPLY TO THIS MESSAGE ***. Disclaimer: This
email is best viewed in HTML. This email was sent from a notification-only
address that cannot accept incoming email.*
*Please confirm that the above mailing address is correct. If not please
email us at addressqa@hp.com <addressqa@hp.com> including your CSO #*

Hello HANDY,

We would like to inform you that the HP Repair Facility has received your
unit and will proceed to repair your product.
➀ Order Created
➁ Packing Material
➌ Product Received
➃ Repaired Product Shipped
➄ Resolution

*NEXT STEPS:*
Once your unit is repaired, you will receive a notification once it is in
transit.

------------------------------

*Please be aware of the following:*

   - To ensure that your case is resolved, if we are unable to duplicate
   your issue we may contact you for further information.
   - If damage is found not covered under warranty or under the repair
   quote that was given, you will be contacted with a new quote.
   - If we are unable to reach you, by phone and/or e-mail within 3
   business days, your unit will be returned to you unrepaired.

To review the status of your order online, please have your CSO#
BLPN0743-01 and email address ready, and visit the HP Case Status Page
<http://www.hp.com/go/csostatus>.

Product Receipt Details:
------------------------------
Model Number:
Z4W83AV
Serial Number:
CND7250448
Receipt Date of Customer Product:
05-03-2018
Expected Delivery Date:
05-24-2018
Receiver ID:
TX80X18
------------------------------

*Sincerely,*
*HP Customer Support*

For additional assistance with your HP product, services or instruction on
upgrading Windows 10 please visit www.support.hp.com
------------------------------

*For more information regarding HP's privacy policies and practices, please
visit our Privacy Statement
<http://www8.hp.com/us/en/privacy/ww-privacy.html>.*


*THIS COMMUNICATION IS PROVIDED FOR INFORMATION PURPOSES ONLY.The only
warranties for HP products and services are set forth in the express
warranty statements accompanying such products and services. Nothing herein
should be construed as constituting an additional warranty. For more
information please visit: Limited Warranty Statement
<http://www8.hp.com/us/en/privacy/limited_warranty.html>.*

*HP shall not be liable for technical or editorial errors or omissions
contained herein. Our advice is strictly limited to the question(s) asked
and is based on the information provided to us, please review the HP
Support Website <http://support.hp.com/us-en?openCLC=true>.*


© *Copyright 2016 HP Development Company, L.P.*

**Pull request merged**